### PR TITLE
fix(docs-infra): generate the `404.html` page correctly

### PR DESCRIPTION
**Pull request by gkalpak**
Previously, the generated `404.html` page did not include a `<body>` tag. In some browsers (such as IE 11), this was causing warnings in the console.

This commit ensures the generated page contains a `<body>` tag. It also fixes the indentation in the generated page.

### CODE REVIEW

File: aio/src/404-body.html [hunk 1]
```diff
@@ -1,52 +1,52 @@
-<!--
-  This content replaces the `<body>` content of `index.html` to generate our custom `404.html` page.
-  The content must visually and structurally resemble the resulting HTML of the main app for not
-  found pages (e.g. https://angular.io/not/existing).
--->
+  <!--
+    This content replaces the `<body>` content of `index.html` to generate our custom `404.html` page.
+    The content must visually and structurally resemble the resulting HTML of the main app for not
+    found pages (e.g. https://angular.io/not/exist.ing).
```
Comment (petebacondarwin) [inline aio/src/404-body.html:4]: Why this change?

**Review by petebacondarwin — APPROVED** [review]

File: aio/src/404-body.html (hunk 1 above)
Comment (gkalpak) [reply aio/src/404-body.html:4]: This 404 page is only used for "file URLs" (i.e. URLs that have a period in their last segments). For non-"file URLs" (such as `/not/existing`), `index.html` will be returned and the in-app `Not Found` page will be shown via JS :grin:
